x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -75,6 +77,47 @@ export const signalRulesAlertType = ({
       } = params;
       // TODO: Remove this hard extraction of name once this is fixed: https://github.com/elastic/kibana/issues/50522
       const savedObject = await services.savedObjectsClient.get('alert', alertId);
+      const ruleStatusSavedObjects = await services.savedObjectsClient.find<
+        IRuleSavedAttributesSavedObjectAttributes
+      >({
+        type: ruleStatusSavedObjectType,
+        perPage: 6, // 0th element is current status, 1-5 is last 5 failures.
+        sortField: 'statusDate',
+        sortOrder: 'desc',
+        search: `${alertId}`,
+        searchFields: ['alertId'],
+      });
+      logger.debug(`ruleStatusSavedObjects: ${JSON.stringify(ruleStatusSavedObjects, null, 4)}`);
```

Comment:
I don't think you want to check this logger.debug in with the stringify. Wouldn't that be a lot of noise for the executor to pump out the whole saved objects?

Was this just for testing things out?

Reply:
Yup I'll delete it. I think I left it in figuring it would be nice to see if running in debug mode. Not a problem for me to delete it though.